src/components/textfield/InputField.scss

```
@@ -10,7 +10,42 @@
     border: 1px solid $gray-300;
     background-color: $color--white;
     outline: none;
+    .inputlayer {
+      .input {
+        border-radius: 8px;
+        padding: 12px 16px;
+        border: 1px solid $gray-300;
+        background-color: $color--white;
+        outline: none;
 
+        @include desktop {
+          width: 720px;
+        }
+        @include tablet {
+          width: 720px;
+        }
+        @include mobile {
+          width: 320px;
+        }
```

desktop과 tablet에서는 같은 너비를 지정하고, mobile 화면에서만 다른 너비를 지정해준다면
```suggestion
          width: 720px;
          
        @include mobile {
          width: 320px;
        }
```
이런식으로 해도 괜찮을 것 같아요! 믹스인 잘 사용해주셔서 뿌듯해요 👍👍👍

---

잘 만들어주신 덕분!!👍👍 바로 수정했습니당 :)